.github/workflows/clp-core-build.yaml

```
@@ -147,6 +154,32 @@ jobs:
+  ubuntu-noble-deps-image:
+    name: "ubuntu-noble-deps-image"
+    if: "needs.filter-relevant-changes.outputs.ubuntu_noble_image_changed == 'true'"
+    needs: "filter-relevant-changes"
+    runs-on: "ubuntu-latest"
+    steps:
+      - uses: "actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683"
+        with:
+          submodules: "recursive"
+
+      - name: "Work around actions/runner-images/issues/6775"
+        run: "chown $(id -u):$(id -g) -R ."
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2046:warning:1:7: Quote this to prevent word splitting

(shellcheck)